app/assets/javascripts/components/overview/course_cloned_modal.jsx

```diff
@@ -146,14 +146,6 @@ const CourseClonedModal = createReactClass({
     return course.cloned_status === 2;
   },
 
-  saveEnabled() {
-    // You must be logged in and have permission to edit the course.
-    // This will be the case if you created it (and are therefore the instructor) or if you are an admin.
-    if (!this.props.currentUser.isAdvancedRole) { return false; }
```

Comment:
I think this first check, to make sure the user would be allowed to save it, is still useful.

Reply:
reverted! It's back now